src/mjolnir/pbfgraphparser.cc

```diff
@@ -1946,58 +1938,65 @@ struct graph_callback : public OSMPBF::Callback {
     return std::string(lua_graph_lua, lua_graph_lua + lua_graph_lua_len);
   }
 
-  virtual void node_callback(const uint64_t osmid,
-                             const double lng,
-                             const double lat,
-                             const OSMPBF::Tags& tags) override {
+  // Handle bike share stations separately
+  void bss_node(const osmium::Node& node) {
```

Comment:
Without a callback-style processing we can move bss processing into a separate step